about.html

```diff
@@ -0,0 +1,182 @@
+        <section id="speakers" class="speakers d-flex flex-column align-items-center py-4">
+            <div class="container">
+                <div class="title-2 pt-4 d-flex flex-column align-items-center text-center">
+                    <h2 class="m-0 fw-bold">Dev Conference and I/O events logo</h2>
+                    <hr />
+                </div>
+                <div class="speaker-wrapper pt-3 pb-4">
+                    <div class="row">
+                        <div class="col-lg-12 text-center text-secondary">
+                            <p>
+                                The logo of our compnay was crafted through the idea from the first Conference.
+                                .After a series of meetings and public voting and final review from the selection
+                                Committe's judging process, the logo from the Naresh Agrawal won the prize.
+                                After consultations with Mr. Agrawal we optimised the logo for each and every Conference
+                                we shall have
+                            </p>
+                            <div class="col-md-8 px-5 border border-3 mx-auto">
+                                <img class="img-fluid" src="./assests/img/devcon5.webp" alt="devcon logo" />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <hr class="border border-1" />
+        <section class="speakers d-flex flex-column align-items-center py-4">
+            <div class="container">
+                <div class="title-2 pt-4 d-flex flex-column align-items-center text-center">
+                    <h2 class="m-0 fw-bold">See the past Devcon and I/O events</h2>
+                    <hr />
+                </div>
+                <div class="speaker-wrapper pt-3 pb-4">
+                    <div class="row">
+                        <div class="col-lg-12 text-center text-secondary">
+                            <p>
+                                Take a look at the last two Devcon events which took place in Apple park
+                            </p>
+                            <div class="past-events d-flex w-100 flex-column px-md-5 flex-md-row text-light">
+                                <div class="event-1 mx-md-3 d-flex align-items-center justify-content-center">
+                                    <div>
+                                        <span>2019</span>
+                                        <p>Dev con 2019 in Paris</p>
+                                    </div>
+                                </div>
+                                <div class="event-2 d-flex align-items-center justify-content-center">
+                                    <div>
+                                        <span>2018</span>
+                                        <p>Dev con 2018 in Paris</p>
+                                    </div>
```

Comment:
- [ ] The bottom section of the About page should have 2 images with text centered and an orange overlay above like the template. Kindly add some images to this box to make this section similar to the template.
 - Your design:
 - 
<img width="1017" alt="image" src="https://user-images.githubusercontent.com/61919617/217872811-12bd0332-0ad6-425c-b7f8-69f70575c369.png">

 - Template:
 - 
<img width="746" alt="image" src="https://user-images.githubusercontent.com/61919617/217872950-8df1d7b5-634a-4a96-82d4-eda3b08b1094.png">